### ISSUE TITLE
error running example.py

### ISSUE THREAD

**Issue by charlesmartin14** [google-research/google-research#33]
I am trying to run the example.py in a conda 2.7 environment

get the following error:

```
 File "demogen/data_util.py", line 73, in input_data
    dataset = prob.dataset(mode)
  File "/Users/charleshmartin/anaconda3/envs/demo2/lib/python2.7/site-packages/tensor2tensor/data_generators/problem.py", line 631, in dataset
    assert data_dir
```

**Comment by uricohen** [comment]
Same for me! 

Perhaps this is a setup issue of tensorflow-datasets? I didn't configure it to download CIFAR, etc.

**Comment by uricohen** [comment]
It seems that [tensor2tensor's problem.dataset()](https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/data_generators/problem.py) has changed and now require the second argument `data_dir` not to be left empty.

I could solve the problem by changing `demogen/data_util.py` as follows:
```
  def input_data():
    """Input function to be returned."""
    prob = problems.problem(problem_name)
    data_dir = '<<<MY_HOME_DIRECTORY>>>/t2t_data/';
    if data == 'image_cifar100':
      dataset = prob.dataset(mode, data_dir, preprocess=augmented)
      if not augmented: dataset = dataset.map(map_func=standardization)
    else:
      dataset = prob.dataset(mode, data_dir)
      dataset = dataset.map(map_func=standardization)

```

and I have previously downloaded the data into this directory using:
```
  t2t-trainer --generate_data --data_dir=~/t2t_data --problem=image_cifar100
  t2t-trainer --generate_data --data_dir=~/t2t_data --problem=image_cifar10
```

**Comment by yidingjiang** [comment]
Thank you for working this out. I am working on potentially easier way to get the dataset for external users.

**Comment by uricohen** [comment]
Thanks, @yidinggiang! But are you sure the data split into training/validation/test set is determenistic?
That is, that the CIFAR10/100 training set I got is the same as the one used to train the models?

**Comment by yidingjiang** [comment]
CIFAR10 and 100 have fixed training and test sets, so I don't see any problems. (There have been some changes to the data pipeline, please pull to get them)
